packages/core/nuxt-theme-module/theme/pages/Home.vue

```diff
@@ -217,27 +253,53 @@ export default {
 <style lang="scss" scoped>
 #home {
   box-sizing: border-box;
+  padding: 0 var(--spacer-sm);
   @include for-desktop {
     max-width: 1240px;
+    padding: 0;
     margin: 0 auto;
   }
 }
-.section {
-  padding: 0 var(--spacer-sm);
+
+.hero {
+  margin: var(--spacer-xl) auto var(--spacer-lg);
+  --hero-item-background-position: center;
+  ::v-deep .sf-link:hover {
+    color: var(--c-white);
+  }
   @include for-desktop {
-    padding: 0;
+    margin: var(--spacer-xl) auto var(--spacer-2xl);
   }
 }
 
 .sf-hero-item {
```

Comment:
shouln't that be a sub class of `hero` ? Otehrwise it will affect all future hero items on this page